### ISSUE TITLE
How to categorize fake news sources?

### ISSUE THREAD

**Issue by bkrall** [politecho/politecho#5]
Think we need some kind of criteria for what fake news is and a spectrum for credibility of each source.  For ex., HuffPo and late night shows are not as well sourced as NYT, The Economist but have the [same weighting](https://github.com/politecho/politecho/blob/master/politecho/scoring.js#L30) of ultra-liberal without being penalized at all for being entertainment (as far as I can tell).  

I also realize this tool's purpose is to identify your bubble, not necessarily how informed you are. So with that in mind, my main objective is to recognize that sources like Breitbart is labeled as fake news (rightfully so, as it's Buzzfeed for conservatives) but these late-night shows are not.  Trying to think of a way to reconcile that. 

This is an amazing project, I want to contribute however possible.

**Comment by PersephoneKarnstein** [comment]
I agree, and was going to post something similar. Specifically, I'm concerned also that because the news sources are grouped into only four categories (ultra-liberal; liberal; conservative; ultra-conservative), you lose nuance as to where sources in the same category compare to each other. There are a few places I found in a quick google search that try to rank sources on a numerical/smoother scale, for example [this Pew study](http://www.journalism.org/2014/10/21/political-polarization-media-habits/), [allsides.com](http://www.allsides.com/bias/bias-ratings), and these [two](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=A2aG9S0AAAAJ&citation_for_view=A2aG9S0AAAAJ:hqOjcs7Dif8C) journal [articles](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=SftrEEMAAAAJ&cstart=20&sortby=pubdate&citation_for_view=SftrEEMAAAAJ:4JMBOYKVnBMC). I realize your purpose may not be to make an excruciatingly accurate tool, but rather to give users simply a general idea of their bias, but it would be really cool to see this added. Not to mention that periodically scraping the allsides page, for example, would allow you to vastly expand your source database with minimal grunt work on your end.
